lib/src/generators/openapi3/openapi3.ts

```diff
@@ -333,3 +338,14 @@ function httpMethodToPathItemMethod(
       assertNever(method);
   }
 }
+
+function exampleToOpenApiExampleModel(
+  examples?: Example[]
+): ExamplesSet | undefined {
+  return examples?.reduce<ExamplesSet>((acc, example: Example) => {
+    acc[example.name] = {
+      value: example.value
+    };
+    return acc;
+  }, {});
```

Comment:
We should add a test to cover a contract that generates an example in `openapi3.spec.ts`